core_app/database/streamlined_zipform_functions.py

```
@@ -0,0 +1,444 @@
+        conn.close()
+
+        # Convert to list of dicts
+        contact_list = []
+        for contact in contacts:
+            contact_dict = dict(contact)
+            # Parse additional_info JSON
+            if contact_dict['additional_info']:
+                try:
+                    contact_dict['additional_info'] = json.loads(contact_dict['additional_info'])
+                except:
+                    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)